src/ds++/test/CMakeLists.txt

```diff
@@ -44,6 +44,11 @@ add_scalar_tests(
    DEPS    Lib_dsxx
    )
 
+# tstatomics will use 19 hw threads. Help ctest schedule this job by setting
+# test properties. We don't need to reserve 19 cores, but let's reserve 10
+# (assume 2 hw threads per physical core.)
+set_tests_properties( dsxx_tstatomics PROPERTIES PROCESSORS 10 )
+
```

Comment:
@timmah Did you add `tstatomics` to ds++?  Does it make sense to instruct _ctest_ to reserve 10 HW threads for this test?  Should it be 19?

Reply:
I did that. The number of cores *should* (ha ha!) not affect correctness, so if reserving some cores helps tests execute more smoothly, that *should* be great. The atomics should produce the right answer even in the presence of context switching, resource contention, etc etc, so it might be a more satisfying test if the system is stressed.

Reply:
This test always passes, but I noticed some contention with other MPI-based tests.  I'm not sure this change will help in all cases, but it did in my minimal testing.